### PR TITLE
This branch is to create mobile menu in the Hamburger icon.

### PR DESCRIPTION
**Pull request by tkz96**
Added:

1. Styles and Markup for the Mobile menu icon.
2. Javascript Linters
3. Script to responsive Nav-bar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,24 +26,23 @@
...
-      <div class="nav-element-desktop">
-        <h3 class="elements">Portfolio</h3>
-        <h3 class="elements">About</h3>
-        <h3 class="elements">Contact</h3>
+  
+    <nav class="nav-bar">
+      <div class="nav-mobile">
+        <a class="logo logo-desktop" href="#">My Logo</a>
+        <a class="open-menu" href="#"><i class="fa fa-bars"></i></a>
+        <div class="close-menu hidden"><a href="#"><i class="fa fa-times"></i></a></div>
```
Comment (gemmen29) [inline index.html:34]: Kindly change the ` href ` so when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed and the menu is closed, currently pressing on each link not doing anything.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @tkz96, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (thenarain) [reply index.html:34]: Thanks for the valuable review. I have fixed the changes.

Comment (gemmen29) [inline index.html:34]: [OPTIONAL] Kindly use the same event you use in the close button when pressing on any link as it now the scroll of the page disappear.

**Review by gemmen29 — APPROVED** [review]
Hi @tkz96, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**